src/sentry/filters/preprocess_hashes.py

```diff
@@ -11,3 +16,153 @@
 
 def get_raw_cache_key(project_id, event_id):
     return 'e:raw:{1}:{0}'.format(project_id, event_id)
+
+
+def is_invalid_stack(frames):
```

Comment:
In the future while we're still running experiments to test the approach, it probably makes sense to include these self-contained bits in the test script.